tktermwidget/tkterm.py

```diff
@@ -37,18 +40,20 @@ def __init__(self, master=None, **kwargs):
         Scrollbar.__init__(self, master=master, **kwargs)
 
     def set(self, first: int, last: int):
+        """Set the Scrollbar"""
         if float(first) <= 0.0 and float(last) >= 1.0:
             self.grid_remove()
         else:
             self.grid()
         Scrollbar.set(self, first, last)
 
+
 class Terminal(Frame):
     """A terminal widget for tkinter applications
 
     Args:
```

Comment:
Id make an args markdown page that goes into more detail.

Reply:
I forget to use black, Sorry,